services/checks/checks.go

```diff
@@ -71,6 +75,11 @@ const (
 
 	alertsPrefix        = "/stt/"
 	maxSupportedVersion = 1
+
+	// limits for running pmm-managed-starlark
+	cpuLimit      = 4         // 4 seconds of CPU time
+	memoryLimit   = 100000000 // 100MB of memory in bytes
```

Comment:
Probably it will be nice to have units in a variable name, `sec` and `bytes`

Also 100MB it is 104857600 Bytes, you can write it as:
`memoryLimitBytes   = 100 * 1024 * 1024`